components/List/src/List.tsx

```diff
@@ -1,33 +1,28 @@
-import React from 'react';
-import { List as MaterialList } from '@material-ui/core';
-import BaseProps from '@gemeente-denhaag/baseprops';
+import React, { HTMLAttributes } from 'react';
 import { ListSubheaderProps } from './ListSubheader';
-import { list_classes as classes } from './bem-mapping';
+import clsx from 'clsx';
 
-import './mui-override.scss';
-import './list-item.scss';
-import './list-item-secondary-action.scss';
-import './list-item-icon.scss';
-import './list-item-text.scss';
-import './list-subheader.scss';
+import './list.scss';
 
-export interface ListProps extends Omit<BaseProps, 'classes'> {
+export interface ListProps extends HTMLAttributes<HTMLDivElement> {
   /**
    * The content of the subheader, normally `ListSubheader`.
    */
-  subheader: React.ReactElement<ListSubheaderProps>;
+  subheader?: React.ReactElement<ListSubheaderProps>;
 }
 
 /**
  * Primary UI component for user interaction
  */
-export const List: React.FC<ListProps> = (props: ListProps) => {
-  const materialProps = { subheader: props.subheader };
-
+export const List: React.FC<ListProps> = ({ subheader, ...props }: ListProps) => {
+  const rootClassNames = clsx('denhaag-list__wrapper', {
```

Comment:
This is overwriting the `props.className`, so users are unable to use that property now.